Sources/LLReport_Types.pas

```diff
@@ -139,6 +141,7 @@   TEnumTranslator = class
       class function TranslateDebugFlags(DebugFlags: TLlDebugFlags): Integer; static;
       class function TranslateTableColoring(TableColoring: TLlTableColoring): Integer;
       class function TranslateLanguage(Language: TLlLanguage): Integer;
+      class function TranslateDictionryType(DictionaryType: TllDictionaryType): Integer; static;
```

Comment:
Small typo in method name, should be "Dictionary" :)

Reply:
I removed it, will commit it again wenn I integrated TDictionary. Commit is then not cluttered with different functionalities